src/Generator/FileGenerator.php

```diff
@@ -336,7 +336,7 @@ public function setClasses(array $classes)
 
     /**
      * @param  string $name
-     * @return ClassGenerator
+     * @return ClassGenerator|bool
```

Comment:
As mentioned in comments below, a type change is a BC break - we should rather identify and fix this edge case scenario, by introducing a fallback or a crash.

Reply:
```php
    /**
     * @param string $name
     *
     * @return ClassGenerator
     * @throws RuntimeException
     */
    public function getClass($name = null)
    {
        if ($name === null) {
            reset($this->classes);
            $class = current($this->classes);
            if (false === $class) {
                throw new RuntimeException('No class is set');
            }

            return $class;
        }

        if (false === array_key_exists($name, $this->classes)) {
            throw new RuntimeException(sprintf('Class %s is not set', $name));
        }

        return $this->classes[(string)$name];
    }
```

I'm thinking about something like this.
Would you prefer a more specific exception, like `ClassNotFoundException` (extending RuntimeException)?

Reply:
If it is possible, a more precise exception is always preferable. It can extend `RuntimeException`, but it already makes everything much cleaner.

Reply:
Alright, I added the checks and exception and switched base to develop.